### ISSUE TITLE
Readme, description: Tell people finding this to also check PeerTube out?

### ISSUE THREAD

**Issue by tuxayo** [Gargron/cobalt#1]
That could only help :)

https://www.patreon.com/posts/update-and-15635446
> someone who was already working on a similar concept, PeerTube, has learned of this and decided to make PeerTube speak the same federation protocol as Mastodon. Now I don't think I need to continue Cobalt.

**Comment by ghost** [comment]
I liked Cobalt's UI much more than I like Peertube's so that bummed me a bit but I guess it could be unproductive to develop two pieces of software with the same features.

**Comment by zcdunn** [comment]
@Lionirdeadman Agreed. I like Cobalt's UI much more, too. Maybe the frontend could be merged with PeerTube?

**Comment by Chocobozzz** [comment]
There will be a new design in PeerTube: https://github.com/Chocobozzz/PeerTube/issues/134

But the backend is an API REST so if you prefer another front-end you are free to develop it :). I'll document the API when we'll release the beta (on march 2018).
